magento/magento2
# Fixed Model box overlapping issue

### Description (*)

1. Model box popup text overlapping on the close icon 
2. Reproduce on 2.3

### Fixed Issues (if relevant)


This is the Direct PR owing to small design change.

### Manual testing scenarios (*)

1. Admin - Catalog - category - click on  "All Store Views" dropdown select "Default Store View"
2. Then you can view the Model box popup 
3. Actual result: http://prntscr.com/ml2grg
4. Expected result: http://prntscr.com/ml2g2j


### Contribution checklist (*)
